dScripts/02_server/Enemy/General/BaseEnemyApe.cpp

```diff
@@ -126,7 +126,7 @@ void BaseEnemyApe::StunApe(Entity* self, bool stunState) {
 
 		auto* skillComponent = self->GetComponent<SkillComponent>();
 		if (skillComponent != nullptr) {
-			skillComponent->Interrupt();
```

Comment:
Same here

Reply:
This one is replaced by a reset, which actually resets the ape upon waking up (even though it shouldn't have skills active at that point) as opposed to interrupt which does not do anything because the ape is stun immune